akka-stream/src/main/scala/akka/stream/javadsl/Flow.scala

```diff
@@ -1942,4 +1936,12 @@ abstract class RunnableGraph[+Mat] extends Graph[ClosedShape, Mat] {
    * Transform only the materialized value of this RunnableGraph, leaving all other properties as they were.
    */
   def mapMaterializedValue[Mat2](f: function.Function[Mat, Mat2]): RunnableGraph[Mat2]
+
+  override def withAttributes(attr: Attributes): RunnableGraph[Mat]
+
+  override def addAttributes(attr: Attributes): RunnableGraph[Mat] =
+    withAttributes(module.attributes and attr)
+
+  override def named(name: String): RunnableGraph[Mat] =
```

Comment:
hm, shouldn't named be addAttributes actually? Now a `named` would remove all other attributes, such as dispatchers etc